src/SFA.DAS.CandidateAccount.Domain/Application/AdditionalQuestion.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace SFA.DAS.CandidateAccount.Domain.Application;
+
+public class AdditionalQuestion
+{
+    public Guid Id { get; init; }
+    public Guid CandidateId { get; init; }
+    public Guid ApplicationId { get; init; }
+    public string? QuestionId { get; init; }
+    public string? Answer { get; init; }
+
+    public static implicit operator AdditionalQuestion(AdditionalQuestionEntity source)
+    {
+        return new AdditionalQuestion
+        {
+            Id = source.Id,
+            Answer = source.Answer,
+            ApplicationId = source.ApplicationId,
+            QuestionId = source.QuestionId,
```

Comment:
Can QuestionId be renamed to QuestionText please? on this entity and the table

Reply:
renamed it